chrome/content/zotero/xpcom/intl.js

```diff
@@ -85,6 +85,30 @@ Zotero.Intl = new function () {
 				this.strings[regexpResult[1]] = regexpResult[2];
 			}
 		}
+		
+		// Automatically register l10n sources for enabled plugins.
+		
+		// A Fluent file located at
+		//   [plugin root]/locale/en-US/make-it-red/strings.ftl
+		// could be included in an XHTML file as
+		//   <link rel="localization" href="make-it-red/strings.ftl"/>
```

Comment:
Zotero uses the extra "zotero" subdirectory ("make-it-red" here), but that's just a historical artifact — we wouldn't want to recommend that for plugins. So:

1) `make-it-red/strings.ftl` → `make-it-red.ftl`

2) What happens without the subdirectory? If you just use `href="make-it-red.ftl"`, will it work as long as there's no conflicting file in another `L10nFileSource`?

Reply:
Sure, that'd be great.